dt/bgp/edpm/deployment/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../../lib/dataplane/deployment
+
+resources:
+  - nova_custom.yaml
```

Comment:
See this comment: https://github.com/openstack-k8s-operators/architecture/pull/193/files#r1590951076